.vscode/launch.json

```
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Web server",
+            "type": "go",
+            "request": "attach",
+            "port": 12345,
+            "host": "192.168.9.2",
```

Once you add the port forwarding to run.sh, you should be able to change this to localhost
```suggestion
            "host": "localhost",
```

---

Done.